.github/workflows/phpunit.yml

```diff
@@ -54,6 +56,15 @@ jobs:
         id: composer-cache
         run: echo "::set-output name=dir::$(composer config cache-files-dir)"
 
+      - name: Set Minimum PHP 8.0 Versions
+        run: composer require phpunit/phpunit:^9.4 --no-interaction --no-progress --no-update
+        if: matrix.php >= 8
+
+      - name: Set Minimum Laravel ${{ matrix.laravel }} Versions
+        run: |
+          composer require "illuminate/contracts:${{ matrix.laravel }}" --no-interaction --no-progress --no-update
+          composer require "vlucas/phpdotenv:${{ matrix.dotenv }}" --no-interaction --no-progress --no-update
```

Comment:
if you would add `--dev` here we would eliminate one warning (`vlucas/phpdotenv is currently present in the require-dev key and you ran the command without the --dev flag, which would move it to the require key.`)

Reply:
@eschricker I added it for us.

Reply:
The `--dev` is just needed on phpdotenv not on illuminate/contracts

Reply:
> The `--dev` is just needed on phpdotenv not on illuminate/contracts

Ok, I already remove it from contracts. You're welcome to close the PR if this is was missing :)